### ISSUE TITLE
Link to currency code table no longer exists

### ISSUE THREAD

**Issue by jacobmischka** [mdn/content#15620]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat

### What information was incorrect, unhelpful, or incomplete?

The [ISO 4217 currency code list](https://www.currency-iso.org/en/home/tables/table-a1.html) links redirect to a landing page for some organization instead of a table of currency codes.

### What did you expect to see?

A table of currency codes.

### Do you have any supporting links, references, or citations?

[ISO 4217 currency code list](https://www.currency-iso.org/en/home/tables/table-a1.html)

### Do you have anything more you want to share?

I would like to post a PR instead of an issue, but I'm not sure a reliable source of this information. That's why I came here! :slightly_smiling_face:

Thank you and all other editors for your help and assistance!

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/intl/numberformat/numberformat`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/intl/numberformat/numberformat/index.md
* Last commit: https://github.com/mdn/content/commit/6f1ecf44bdbbc6a0bf8f8c6be0a6d241dec7b22e
* Document last modified: 2022-04-12T05:38:44.000Z

</details>

**Comment by teoli2003** [comment]
I found this weird the first time I saw the page. (Six Group is a large financial institution, but weird).
But it looked like ISO tasked them to maintain the list, so it looks like the authoritative place.

But maybe we should use an anchor that actually open the form. What about using this link? https://www.six-group.com/en/products-services/financial-information/data-standards.html#scrollTo=currency-codes
